crates/solver-discovery/src/implementations/offchain/_7683.rs

```diff
@@ -525,7 +525,10 @@ impl Eip7683OffchainDiscovery {
 					.input_settler_compact_address
 					.clone()
 					.unwrap_or_else(|| {
-						panic!("No input settler compact address found for chain ID {}", origin_chain_id);
+						panic!(
```

Comment:
Not sure if you already made the change, but just didn't push. However, let's not forget this one please..